WordPress/src/main/res/values/strings.xml

```diff
@@ -3340,14 +3340,14 @@
     <string name="mlp_network_error_subtitle">Tap retry when you\'re back online or create a blank page using the button below.</string>
 
     <!-- Home Page Picker -->
-    <string name="hpp_title">Choose a design</string>
+    <string name="hpp_title">Choose a theme</string>
     <string name="hpp_subtitle">Pick your favorite homepage layout. You can edit and customize it later.</string>
     <string name="hpp_choose_button">Choose</string>
     <string name="hpp_choose_and_create_site">Create site</string>
     <string name="hpp_error_title">Layouts not available while offline</string>
     <string name="hpp_error_subtitle">Tap retry when you\'re back online.</string>
     <string name="hpp_retry_error">Please Check your internet connection and retry.</string>
-    <string name="hpp_choose_error">There was an error while selecting the design.</string>
+    <string name="hpp_choose_error">There was an error while selecting the theme.</string>
```

Comment:
This wasn't part of the task, but I guess this makes sense to change for consistency. If we want to leave it as it was, we can revert this back. Wdyt?

Reply:
Thank you for not missing this, I'd say we go forward with including this change in the merge of this PR 🙇